.github/CONTRIBUTING.md

```diff
@@ -290,3 +300,14 @@ Git can append the `Signed-off-by` line to your commit messages. Simply
 provide the `-s` or `--signoff` option when performing a `git commit`.
 For more information about writing commit messages, visit [How to Write
 a Git Commit Message](https://chris.beams.io/posts/git-commit/).
+
+#### Co-Authored By
+If someone else had part in your PR, please add the following to the commit:
+`Co-authored-by: Name <gitregistered@email.adress>`
+This is useful if their authorship was lost during squashing, rebasing, etc.,
+but may be used in any situation where there are co-authors.
+
+The e-mail adress used here should be the same as on the github profile of said user.
```

Comment:
Some nits:
- I would use "email" instead of "e-mail".
- github -> GitHub
- In "does not have it's email adress public", "it's" is outright wrong and "its" sounds weird. Use "their". Also, typo: adress -> address

Reply:
- I think that can be debated and honestly don't know what the current formalised language guidelines are on e-mail vs email.... Source?
- Will do!
- THANK YOU, I totally forgot singular `their` was a thing if used with a unspecified antecedent.

Reply:
> * I think that can be debated and honestly don't know what the current formalised language guidelines are on e-mail vs email.... Source?

In my particular experience, either is acceptable, but "email" is the more common version these days. I don't have a particular source. If you do a Google search for it, you'll see lots of major places using "email". Wikipedia uses "email": https://en.wikipedia.org/wiki/Email

Reply:
@rlaager 
When it comes to sources:
https://writingexplained.org/e-mail-or-email

Both are perfectly fine...

Reply:
i've decided to go for email instead, simply because the rest of the document uses email instead of e-mail and consistency is a thing...